docs/src/main/sphinx/admin/properties-node-scheduler.rst

```diff
@@ -61,6 +61,17 @@ if set too high.
 
     Only applies for ``uniform`` :ref:`scheduler policy <node-scheduler-policy>`.
 
+``node-scheduler.max-unacknowledged-splits-per-task``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``2000``
+
+Maximum number of splits that are either queued on the coordinator, but not yet sent or confirmed to have been received by
```

Comment:
Does increasing this value have any significant implications for memory usage on the coordinator ?
I assume this change implies that every table scan can now queue up 4X more splits on the coordinator.

Reply:
This is limited by `max-splits-per-node` which is still set as default to 100. A higher number is currently used when we adjust queue size if splits are processed faster than assigned.